src/More/About.spec.tsx

```diff
@@ -1,44 +1,57 @@
 import React from "react"
 import { Linking } from "react-native"
-import { render, fireEvent } from "@testing-library/react-native"
-import {
-  getApplicationName,
-  getVersion,
-  getBuildNumber,
-} from "react-native-device-info"
+import { render, fireEvent, wait, act } from "@testing-library/react-native"
 
 import AboutScreen from "./About"
-
-jest.mock("react-native-device-info", () => {
-  return {
-    getApplicationName: jest.fn(),
-    getBuildNumber: jest.fn(),
-    getVersion: jest.fn(),
-  }
-})
+import * as NativeModule from "../gaen/nativeModule"
```

Comment:
would it make sense to introduce another native module here? This doesnt seem related to gaen specifically, and it might be nice to allow them to be treated separately.  so something like
```
import * as DeviceInfo from "./deviceInfoNativeModule"
```